TCAT/Views/Circle.swift

```diff
@@ -36,10 +38,10 @@ class Circle: UIView {
                 
                 backgroundColor = .white
                 layer.borderColor = color.cgColor
-                layer.borderWidth = 1.0
+                layer.borderWidth = 2.5
                 
                 let solidCircle = CALayer()
-                solidCircle.frame = CGRect(x: 0, y: 0, width: 12, height: 12)
+                solidCircle.frame = CGRect(x: 0, y: 0, width: 6, height: 6)
```

Comment:
I worked with Mihir all on the Circle file here, I'll make sure I take the enum value and good stuff here and merge with mine